src/Components/navbar/navbar.jsx

```diff
@@ -20,11 +23,11 @@ function Navbar() {
                         <div class="icon-badge">
                             <button type="button" class="icon-button" id="cart-button">
                                 <span>
-                                  <img class="icon-img" src="/Images/icons8-trolley-cart-64.png" />
+                                <Link to="/cart-page"> <img class="icon-img" src="/Images/icons8-trolley-cart-64.png" /></Link>
```

Comment:
![image](https://user-images.githubusercontent.com/67889207/168415390-590481c2-2d28-471e-aacb-55bd4e30ed12.png)
Image is not being loaded.